Makefile

```diff
@@ -33,6 +33,14 @@ test-e2e-connect: ## Run e2e tests using Connect (requires OP_CONNECT_TOKEN and
 	@echo "[INFO] Running e2e tests with Connect authentication..."
 	@sh -c 'unset OP_SERVICE_ACCOUNT_TOKEN; OP_CONNECT_TOKEN="$(OP_CONNECT_TOKEN)" OP_CONNECT_HOST="$(OP_CONNECT_HOST)" TF_ACC=1 go test -v ./test/e2e/... -timeout 30m'
 
+test-e2e-account: ## Run e2e test using account methodology with Touch ID (requires OP_ACCOUNT, manual only). Creates multiple items to verify biometrics is only prompted once.
+	@test -n "$(OP_ACCOUNT)" || (echo "[ERROR] OP_ACCOUNT environment variable is not set."; exit 1)
```

Comment:
Let's add the same check for `OP_TEST_VAULT_NAME` env var, so it doesn't run tests if it's not set.